lib/blueprinter/base.rb

```diff
@@ -10,18 +10,74 @@
 
 module Blueprinter
   class Base
+    # Specify a field or method name used as an identifier. Usually, this is
+    # something like :id
+    #
+    # @param method [Symbol] the method or field used as an identifier that you
+    #   want to set for serialization.
+    # @param name [Symbol] to rename the identifier key in the JSON
+    #   object. Defaults to method given.
+    # @param serializer [AssociationSerializer,PublicSendSerializer]
+    #   Kind of serializer to use.
+    #   Either define your own or use Blueprinter's premade serializers.
+    #   Defaults to PublicSendSerializer
+    #
+    # @example Specifying a uuid as an identifier.
+    #   class UserBlueprint < Blueprinter::Base
+    #     identifier :uuid
+    #     # other code
+    #   end
+    #
+    # @return [Field] A Field object
     def self.identifier(method, name: method, serializer: PublicSendSerializer)
       view_collection[:identifier] << Field.new(method, name, serializer)
     end
 
+    # Specify a field or method name to be included for serialization.
+    # Takes a required method and an option.
+    #
+    # @param method [Symbol] the field or method name you want to include for
+    #   serialization.
+    # @param options [Hash] options to overide defaults.
+    # @option options [AssociationSerializer,PublicSendSerializer] :serializer
+    #   Kind of serializer to use.
+    #   Either define your own or use Blueprinter's premade serializers. The
+    #   Default serializer is AssociationSerializer
+    # @option options [Symbol] :name Use this to rename the method. Useful if
+    #   if you want your JSON key named differently than your object's field
+    #   or method name.
+    #
+    # @example Specifying a user's first_name to be serialized.
+    #   class UserBlueprint < Blueprinter::Base
+    #     field :first_name
+    #     # other code
+    #   end
+    #
+    # @return [Field] A Field object
     def self.field(method, options = {})
       name = options.delete(:name) || method
       serializer = options.delete(:serializer) || AssociationSerializer
```

Comment:
NB: So your documentation drew attention to this - I think this should default to `PublicSendSerializer` vs `Association` - which is my fault but I don't know why I had it this way. I think we should clean up this logic in a separate PR?

Reply:
Ok, issue created here #30